### PR TITLE
after npm audit fix and npm update

### PR DESCRIPTION
**Pull request by scytacki**
result of running 
`npm audit fix`
`npm update`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,60 +66,60 @@
...
     "@types/enzyme-adapter-react-16": "^1.0.6",
-    "@types/jest": "^26.0.19",
-    "@types/react": "^16.14.2",
-    "@types/react-dom": "^16.9.10",
-    "@typescript-eslint/eslint-plugin": "^4.9.1",
-    "@typescript-eslint/parser": "^4.9.1",
-    "autoprefixer": "^10.1.0",
+    "@types/jest": "^26.0.22",
+    "@types/react": "^16.14.5",
+    "@types/react-dom": "^16.9.12",
```
Comment (kswenson) [inline package.json:78]: Given that we have React 17 in `dependencies`, we should really be using the React 17 version of the `@types/react*`, but that's a preexisting condition so feel free to kick that can down the road if you're so inclined.

File: package.json [hunk 2]
```diff
@@ -66,60 +66,60 @@
...
+    "@typescript-eslint/parser": "^4.22.0",
+    "autoprefixer": "^10.2.5",
     "babel-core": "^6.26.3",
     "babel-jest": "^26.6.3",
     "clean-webpack-plugin": "^3.0.0",
-    "copy-webpack-plugin": "^6.4.0",
-    "css-loader": "^5.0.1",
+    "copy-webpack-plugin": "^6.4.1",
+    "css-loader": "^5.2.1",
     "cypress": "^5.6.0",
```
Comment (kswenson) [inline package.json:87]: There aren't very many tests so we could probably go to the latest version of cypress.

**Review by kswenson — APPROVED** [review]
👍 Looks fine -- there are a couple of additional updates that are worth doing at some point, but whether or not now is that time I leave to your discretion. 😉

File: package.json (hunk 1 above)
Comment (scytacki) [reply package.json]: Good point. Done.

File: package.json (hunk 2 above)
Comment (scytacki) [reply package.json]: Good point. Done.

**Pull request merged**